sssom/cli.py

```diff
@@ -89,6 +91,32 @@
     help="Fields.",
 )
 
+predicate_filter_option = click.option(
+    "-F",
+    "--mapping-predicate-filter",
+    multiple=True,
+    help="A list of predicates or a file path containing the list of predicates to be considered.",
+)
+
+
+def _get_list_of_predicate_iri(predicate_filter: tuple) -> list:
```

Comment:
we should have no methods in cli that are not 100% cli() related. This method here is a bit on the fence. don't we do all the argument parsing usually in io.py? Not sure whats best here.

Reply:
moved to io.py